src/lib/core/testing/jasmine-matchers.ts

```diff
@@ -10,21 +10,35 @@
  * Collection of useful custom jasmine matchers for tests.
  */
 export const customMatchers: jasmine.CustomMatcherFactories = {
+  /** Matcher verifying that the element has the expected role attribute. */
   toBeRole: () => {
     return {
       compare: function (element: Element, expectedRole: string) {
-        const result: jasmine.CustomMatcherResult = {pass: false};
-        const actualRole = element.getAttribute('role');
-
-        result.pass = actualRole === expectedRole;
-        result.message = `Expected role for ${element.tagName} to be ${expectedRole}`;
-
-        if (!result.pass) {
-          result.message += ` but was ${actualRole}`;
-        }
+        return checkElementAttribute(element, 'role', expectedRole);
+      }
+    };
+  },
 
-        return result;
+  /** Matcher verifying that the element has the expected aria label. */
+  toHaveAriaLabel: () => {
```

Comment:
Do we really need an extra matcher just for the `aria-label`? It seems like we can end up with having a matcher for every attribute.